config/deploy.ignore

```diff
@@ -1,23 +1,36 @@
+.circleci/
+.DS_store
+.editorconfig
+.eslintrc.json
 .git/
-selenium*
-fabfile.py*
+.github/
+.gitignore
+.prettierrc
+.stylelintrc
+.vscode
 *.pyc
+CHANGELOG.md
+CODE_OF_CONDUCT.md
+config/secrets.py.example
+CONTRIBUTING.md
 data/
 docs/
 env/
+fabfile.py*
 git_modules/
 grunt-tasks/
-node_modules/
 Gruntfile.js
+icon-licenses/
 Makefile
+node_modules/
+package-lock.json
 package.json
 README.md
-session.db
+run.py
 screenshots/
+selenium*
+session.db
 tests/
-icon-licenses/
-CHANGELOG.md
-CONTRIBUTING.md
-webcompat.sh
-uploads/
 tmp/
+uploads/
+webcompat.sh
```

Comment:
This could be removed, it doesn't exist anymore.

Reply:
ok I'll do remove this one from the deploy and double check it's not on the server too.